prepare_oim/roles/deploy_containers/pcs/tasks/validate_high_availability.yml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2025 Dell Inc. or its subsidiaries. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+---
+
+- name: Include high_availability_config.yml
+  block:
+    - name: Include high_availability_config.yml
+      ansible.builtin.include_vars: "{{ high_availability_config_path }}"
+      register: high_availability_config
+      tags: init
+  rescue:
+    - name: Failed to include high_availability_config.yml
+      ansible.builtin.fail:
+        msg: "{{ high_availability_config_syntax_fail_msg }} Error: {{ high_availability_config.message }}"
+
+- name: Include network_spec.yml
+  block:
+    - name: Include network_spec file
+      ansible.builtin.include_vars: "{{ network_spec }}"
```

Comment:
this include is done in prepare_oim_validation. why its required again? we can use hostvars['localhost'] to fetch values